tests/baselines/reference/api/typescript.d.ts

```diff
@@ -9825,6 +9829,18 @@ declare namespace ts {
      * @param context A lexical environment context for the visitor.
      */
     function visitEachChild<T extends Node>(node: T | undefined, visitor: Visitor, context: TransformationContext, nodesVisitor?: typeof visitNodes, tokenVisitor?: Visitor): T | undefined;
+    function emitDeclarationsForProject(projectPath: string, files: string[] | undefined, options: CompilerOptions, host: CompilerHost): string;
+    function createIsolatedDeclarationsEmitter(rootDir: string, options: CompilerOptions): (file: string, host: CompilerHost) => {
+        output?: undefined;
+        diagnostics?: undefined;
+    } | {
+        output: string;
+        diagnostics: Diagnostic[];
+    };
+    function emitDeclarationsForFile(sourceFile: SourceFile, options: CompilerOptions): {
+        code: string;
+        diagnostics: Diagnostic[];
+    };
```

Comment:
Do these need to be exported?

Reply:
I don't belive we took a final decision if these will be part of the public API. Without exposing these there will be o way to do a fast declaration emit using TSC.